Makefile

```diff
@@ -62,5 +63,15 @@ Gemfile.lock: Gemfile
 up: Dockerfile.built
 	docker run --rm -p 80:4567 $(IMAGE)
 
-test:
-	docker run --rm -e TALKTOME_EMAIL_DEFAULT_FROM=from@talktome.com $(IMAGE) bundle exec rake test
+test: Dockerfile.built
+	docker run --rm -e TALKTOME_EMAIL_DEFAULT_FROM=from@talktome.com $(IMAGE) sh -c "bundle install && bundle exec rake test"
+
+################################################################################
+### Gem Management
+###
+
+talktome.gem:
+	docker run -t --rm -v ${PWD}/:/app -w /app ruby bash -c "gem build talktome.gemspec -o talktome.gem"
```

Comment:
There is a `bundle exec rake gem` available on all projects, that puts the gem in `pkg/...`. We need to use that one.